docs-md/operate-and-deploy/installation/installing.md

```diff
@@ -3,144 +3,222 @@ layout: page
 title: Install ksqlDB
 tagline: Install ksqlDB on-premises
 description: Learn how to install ksqlDB on-premises
-keywords: ksql, install, on-prem
+keywords: ksql, install, docker, docker-compose, container, docker image, on-prem
 ---
 
-ksqlDB is a component of {{ site.cp }} and the ksqlDB binaries are located
-at [Confluent Downloads](https://www.confluent.io/download/)
-as a part of the {{ site.cp }} bundle.
+ksqlDB and Docker containers
+----------------------------
 
-ksqlDB must have access to a running {{ site.aktm }} cluster, which can
-be in your data center, in a public cloud, {{ site.ccloud }}, etc.
+You can run ksqlDB locally by using Docker containers, and you can define a
+ksqlDB application by creating a *stack* of containers. A stack is a group of
+containers that run interrelated services. For more information on stacks, see
+[Describing Apps Using Stack Files](https://docs.docker.com/get-started/part4/#describing-apps-using-stack-files).
 
-Docker support
---------------
-
-You can deploy ksqlDB by using
-[Docker containers](install-ksqldb-with-docker.md).
-Starting with {{ site.cp }} 4.1.2, Confluent maintains images at
-[Docker Hub](https://hub.docker.com/r/confluentinc/ksqldb-server/).
-To start ksqlDB containers in configurations like "ksqlDB Headless
-Server" and "Interactive Server with Interceptors", see
-[Docker Configuration Parameters](https://docs.confluent.io/current/installation/docker/config-reference.html).
-
-Watch the
-[screencast of Installing and Running KSQL](https://www.youtube.com/embed/icwHpPm-TCA).
-
-Supported Versions and Interoperability
----------------------------------------
-
-You can use ksqlDB with compatible {{ site.cp }} and {{ site.aktm }}
-versions.
+The minimal ksqlDB stack has containers for {{ site.aktm }}, {{ site.zk }}, and
+ksqlDB Server. More sophisticated ksqlDB stacks can have {{ site.sr }},
+{{ site.kconnect }}, and other third-party services, like Elasticsearch.
 
-|    ksqlDB version     | {{ site.version }} |
-| --------------------- | ------------------ |
-| Apache Kafka version  | 0.11.0 and later   |
-| {{ site.cp }} version | > 3.3.0 and later  |
+Stacks that have {{ site.sr }} can use Avro-encoded events in ksqlDB
+applications. Without {{ site.sr }}, your ksqlDB applications can use only JSON
+or delimited formats.  
 
-Installation Instructions
--------------------------
+!!! note
```

Comment:
I cannot find a way to view the rendered output (no link in console output), so cannot view how this `!!! note` renders. Hoping it looks like a normal note and not three exclamation marks with the text.

Reply:
Yes, it'll only render with `mkdocs`, but I doubt it's worth the effort for you to install it. :)